### PR TITLE
Update cards

### PR DESCRIPTION
**Pull request by aymanechilah**
## Description
Update OCR model cards
Remove LayoutLM cards
Add new cards
Add examples

## Motivation and Context
OCR cards were outdated and incomplete

## Types of changes
Update OCR model cards
Remove LayoutLM cards
Add new cards
Add examples

### CODE REVIEW

**Comment by vkocaman** [comment]
@gokhanturer @Damla-Gurbaz  please review and suggest corrections.. then you can ping me for merging. Thanks

**Comment by vkocaman** [comment]
@aymanechilah any update on this ?

**Comment by Damla-Gurbaz** [comment]
Hi @vkocaman , we checked some of them but we haven't completed all of them yet, we plan to complete them today.

**Comment by vkocaman** [comment]
No rush @Damla-Gurbaz 
We dont need it for the release.. you can handle on Monday

File: docs/_posts/kolia1985/2022-06-15-ocr_base_handwritten_v2_en_3_0.md [hunk 1]
```diff
@@ -20,37 +20,87 @@ use_language_switcher: "Python-Scala-Java"
...
+
 ## Predicted Entities
 
 
 
 {:.btn-box}
 <button class="button button-orange" disabled>Live Demo</button>
-<button class="button button-orange" disabled>Open in Colab</button>
-[Download](https://s3.amazonaws.com/auxdata.johnsnowlabs.com/clinical/ocr/ocr_base_handwritten_v2_en_3.4.0_3.0_1655299331379.zip){:.button.button-orange.button-orange-trans.arr.button-icon}
+[Open in Colab]([Open in Colab](https://colab.research.google.com/github/JohnSnowLabs/spark-ocr-workshop/blob/TrainingNotebooks/tutorials/Certification_Trainings/others/SparkOcrImageToTextHandwritten_V2.ipynb){:.button.button-orange.button-orange-trans.co.button-icon}){:.button.button-orange.button-orange-trans.co.button-icon}
```
Comment (Damla-Gurbaz) [inline docs/_posts/kolia1985/2022-06-15-ocr_base_handwritten_v2_en_3_0.md:30]: Colab notebook link is broken. Could you please check?

File: docs/_posts/kolia1985/2021-12-21-text_cleaner_v1_en.md [hunk 2]
```diff
@@ -25,29 +25,109 @@ Model for cleaning image with text. It is based on text detection model with ext
 
 {:.btn-box}
 <button class="button button-orange" disabled>Live Demo</button>
-<button class="button button-orange" disabled>Open in Colab</button>
+[Open in Colab]([Open in Colab](https://colab.research.google.com/github/JohnSnowLabs/spark-ocr-workshop/blob/TrainingNotebooks/tutorials/Certification_Trainings/others/SparkOcrImageCleaner.ipynb){:.button.button-orange.button-orange-trans.co.button-icon}){:.button.button-orange.button-orange-trans.co.button-icon}
```
Comment (Damla-Gurbaz) [inline docs/_posts/kolia1985/2021-12-21-text_cleaner_v1_en.md:28]: Colab notebook link is broken. Could you please check?

File: docs/_posts/kolia1985/2021-12-09-text_detection_v1_en.md [hunk 3]
```diff
@@ -25,23 +25,91 @@ CRAFT: Character-Region Awareness For Text detection, is designed with a convolu
 
 {:.btn-box}
 <button class="button button-orange" disabled>Live Demo</button>
-<button class="button button-orange" disabled>Open in Colab</button>
+[Open in Colab](https://colab.research.google.com/github/JohnSnowLabs/spark-ocr-workshop/blob/master/tutorials/Certification_Trainings/others/SparkOcrImageTextDetection.ipynb){:.button.button-orange.button-orange-trans.co.button-icon}
```
Comment (Damla-Gurbaz) [inline docs/_posts/kolia1985/2021-12-09-text_detection_v1_en.md:28]: Colab notebook link is broken. Could you please check?

File: docs/_posts/kolia1985/2021-12-09-text_detection_v1_en.md [hunk 4]
```diff
@@ -25,23 +25,91 @@ CRAFT: Character-Region Awareness For Text detection, is designed with a convolu
...
+    from pyspark.ml import PipelineModel
+    from sparkocr.transformers import *
+    
+    imagePath = "path to image"
+    image_df = spark.read.format("binaryFile").load(imagePath)
+
+    binary_to_image = BinaryToImage() 
+    binary_to_image.setImageType(ImageType.TYPE_3BYTE_BGR)
+    
+    text_detector = ImageTextDetectorV2 \
```
Comment (Damla-Gurbaz) [inline docs/_posts/kolia1985/2021-12-09-text_detection_v1_en.md:47]: model in name and download parts --> **text_detection_v1**

but model name in the pipeline  --> **image_text_detector_v2**.

File: docs/_posts/aymanechilah/2023-01-17-ocr_base_printed_v2_opt.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+## Description
+
+Ocr base printed model v2 optimized with ONNX for recognise printed text based on Tesseract architecture and printed datasets.
+Tesseract is an optical character recognition engine with open-source code, this is the most popular and qualitative OCR-library. OCR uses artificial intelligence for text search and its recognition on images. Tesseract is finding templates in pixels, letters, words and sentences. It uses two-step approach that calls adaptive recognition. It requires one data stage for character recognition, then the second stage to fulfil any letters, it wasn’t insured in, by letters that can match the word or sentence context.
+
+## Predicted Entities
+
+{:.btn-box}
+<button class="button button-orange" disabled>Live Demo</button>
+[Open in Colab](https://colab.research.google.com/github/JohnSnowLabs/spark-ocr-workshop/blob/master/tutorials/Cards/SparkOcrImageToTextPrinted_V2_opt.ipynb){:.button.button-orange.button-orange-trans.co.button-icon}
```
Comment (Damla-Gurbaz) [inline docs/_posts/aymanechilah/2023-01-17-ocr_base_printed_v2_opt.md:27]: Colab notebook link is broken. Could you please check?

File: docs/_posts/aymanechilah/2023-01-17-ocr_base_printed_v2.md [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+## Description
+
+Ocr base printed model v2 for recognise printed text based on Tesseract architecture and printed datasets.
+Tesseract is an optical character recognition engine with open-source code, this is the most popular and qualitative OCR-library. OCR uses artificial intelligence for text search and its recognition on images. Tesseract is finding templates in pixels, letters, words and sentences. It uses two-step approach that calls adaptive recognition. It requires one data stage for character recognition, then the second stage to fulfil any letters, it wasn’t insured in, by letters that can match the word or sentence context.
+
+## Predicted Entities
+
+{:.btn-box}
+<button class="button button-orange" disabled>Live Demo</button>
+[Open in Colab](https://colab.research.google.com/github/JohnSnowLabs/spark-ocr-workshop/blob/master/tutorials/Cards/SparkOcrImageToTextPrinted_V2.ipynb){:.button.button-orange.button-orange-trans.co.button-icon}
```
Comment (Damla-Gurbaz) [inline docs/_posts/aymanechilah/2023-01-17-ocr_base_printed_v2.md:27]: Colab notebook link is broken. Could you please check?

Comment (aymanechilah) [reply docs/_posts/aymanechilah/2023-01-17-ocr_base_printed_v2.md:27]: solved

File: docs/_posts/aymanechilah/2023-01-17-ocr_base_printed_v2_opt.md (hunk 5 above)
Comment (aymanechilah) [reply docs/_posts/aymanechilah/2023-01-17-ocr_base_printed_v2_opt.md:27]: solved

File: docs/_posts/kolia1985/2021-12-09-text_detection_v1_en.md (hunk 3 above)
Comment (aymanechilah) [reply docs/_posts/kolia1985/2021-12-09-text_detection_v1_en.md:28]: solved

File: docs/_posts/kolia1985/2021-12-21-text_cleaner_v1_en.md (hunk 2 above)
Comment (aymanechilah) [reply docs/_posts/kolia1985/2021-12-21-text_cleaner_v1_en.md:28]: solved

File: docs/_posts/kolia1985/2022-06-15-ocr_base_handwritten_v2_en_3_0.md (hunk 1 above)
Comment (aymanechilah) [reply docs/_posts/kolia1985/2022-06-15-ocr_base_handwritten_v2_en_3_0.md:30]: solved

File: docs/_posts/xyutech/2021-09-04-ocr_table_detection_general_model_en.md [hunk 7]
```diff
@@ -17,39 +17,59 @@ use_language_switcher: "Python-Scala-Java"
...
+Here it is used the CascadeTabNet general model for table detection inspired by https://arxiv.org/abs/2004.12629
 
 ## Predicted Entities
 
 
 
 {:.btn-box}
 <button class="button button-orange" disabled>Live Demo</button>
-<button class="button button-orange" disabled>Open in Colab</button>
+[Open in Colab](https://colab.research.google.com/github/JohnSnowLabs/spark-ocr-workshop/blob/master/tutorials/Certification_Trainings/others/SparkOcrImageTableDetection.ipynb){:.button.button-orange.button-orange-trans.co.button-icon}
```
[the remark this answers is not in the document]
Comment (aymanechilah) [reply docs/_posts/xyutech/2021-09-04-ocr_table_detection_general_model_en.md:30]: solved

File: docs/_posts/kolia1985/2022-06-15-ocr_base_handwritten_v2_en_3_0.md [hunk 8]
```diff
@@ -20,37 +20,87 @@ use_language_switcher: "Python-Scala-Java"
...
+[Download](https://s3.amazonaws.com/auxdata.johnsnowlabs.com/clinical/ocr/ocr_small_handwritten_en_3.3.3_2.4_1645080334390.zip){:.button.button-orange.button-orange-trans.arr.button-icon}
```
[the remark this answers is not in the document]
Comment (aymanechilah) [reply docs/_posts/kolia1985/2022-06-15-ocr_base_handwritten_v2_en_3_0.md:31]: solved

File: docs/_posts/kolia1985/2021-12-09-text_detection_v1_en.md (hunk 4 above)
Comment (aymanechilah) [reply docs/_posts/kolia1985/2021-12-09-text_detection_v1_en.md:47]: solved

File: docs/_posts/kolia1985/2021-12-21-text_cleaner_v1_en.md [hunk 9]
```diff
@@ -25,29 +25,109 @@ Model for cleaning image with text. It is based on text detection model with ext
...
+        .setInputCol("image") \
+        .setOutputCol("text_regions") \
+        .setWithRefiner(True) \
+        .setSizeThreshold(10) \
+        .setScoreThreshold(0.2) \
+        .setTextThreshold(0.2) \
+        .setLinkThreshold(0.3) \
+        .setWidth(500)
+    
+    ocr = ImageToTextV2.pretrained("ocr_base_handwritten", "en", "clinical/ocr") \
```
[the remark this answers is not in the document]
Comment (aymanechilah) [reply docs/_posts/kolia1985/2021-12-21-text_cleaner_v1_en.md:57]: solved.

File: docs/_posts/kolia1985/2022-02-16-ocr_small_printed_en.md [hunk 10]
```diff
@@ -17,68 +17,181 @@ use_language_switcher: "Python-Scala-Java"
...
+        ocr,
+        tokenizer,
+        draw_annotations
+    ])
+
+    hocr_result = hocr_pipeline.transform(image_example_df).cache()
+    display_images(hocr_result, "image_with_annotations", width=1000)
+    print(hocr_result.select("hocr").collect()[0].hocr)
+```
+```scala
```
[the remark this answers is not in the document]
Comment (aymanechilah) [reply docs/_posts/kolia1985/2022-02-16-ocr_small_printed_en.md:83]: solved

File: docs/_posts/xyutech/2021-09-04-ocr_table_detection_general_model_en.md [hunk 11]
```diff
@@ -58,9 +78,26 @@ val tableDetector = ImageTableDetector
 .pretrained("general_model_table_detection_v2", "en", "clinical/ocr")
 .setInputCol("image")
 .setOutputCol("table regions")
+
+val results = pipeline
```
[the remark this answers is not in the document]
Comment (aymanechilah) [reply docs/_posts/xyutech/2021-09-04-ocr_table_detection_general_model_en.md:82]: solved

**Pull request closed without merging**